lib/jnpr/junos/utils/sw.py

```diff
@@ -783,8 +789,8 @@ def _progress(report):
                     "NSSU: installing software ... please be patient ...")
                 return self.pkgaddNSSU(remote_package,
                                        dev_timeout=timeout, **kwargs)
-            elif self._multi_RE is False:
-                # simple case of device with only one RE
+            elif self._multi_RE is False or all_re is False:
```

Comment:
@stacywsmith I understand with this change we dont have any code coverage issue, but can we still add a UT.